tests_pytest/_fw_tests_common_base/fusing/base_fusing_info_generator_test.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2025 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Callable, Any
+
+import copy
+
+import abc
+
+import pytest
+
+
+from model_compression_toolkit.core import QuantizationConfig, FrameworkInfo
+from model_compression_toolkit.core.common import BaseNode
+from model_compression_toolkit.core.common.framework_implementation import FrameworkImplementation
+from model_compression_toolkit.core.common.fusion.fusing_info import FusingInfo
+from model_compression_toolkit.core.common.fusion.fusing_metadata_wrapper import FusingMetadataWrapper
+from model_compression_toolkit.core.common.graph.edge import EDGE_SOURCE_INDEX, EDGE_SINK_INDEX
+from model_compression_toolkit.core.graph_prep_runner import graph_preparation_runner
+from tests_pytest._test_util.tpc_util import minimal_cfg_options
+
+
+class BaseFusingInfoGeneratorTest(abc.ABC):
+
+    fw_impl: FrameworkImplementation
```

Comment:
why not use Mixin?
(not sure it is possible, I'm still trying to understand this mechanism)

Reply:
I'm using it. In the concrete class.